server/src/main/java/org/apache/iotdb/db/engine/compaction/CompactionUtils.java

```diff
@@ -359,6 +359,22 @@ private static void updateOneTargetMods(
     targetFile.getModFile().close();
   }
 
+  public static void removeCompactionModification(
+      List<TsFileResource> selectedSeqTsFileResourceList,
+      List<TsFileResource> selectedUnSeqTsFileResourceList,
+      String fullStorageGroupName) {
+    try {
+      for (TsFileResource seqFile : selectedSeqTsFileResourceList) {
+        ModificationFile.getCompactionMods(seqFile).remove();
+      }
+      for (TsFileResource unseqFile : selectedUnSeqTsFileResourceList) {
+        ModificationFile.getCompactionMods(unseqFile).remove();
+      }
+    } catch (IOException e) {
+      logger.error("{} cannot remove merging modification ", fullStorageGroupName, e);
```

Comment:
```suggestion
      logger.error("{} cannot remove compaction modification ", fullStorageGroupName, e);
```

Also print the mod file name

Reply:
resolved